src/js/tabs/osd.js

```diff
@@ -1172,7 +1172,14 @@ OSD.loadDisplayFields = function() {
                 return `1234${FONT.symbol(SYM.MAH)}/${unit}`;
             },
         },
-
+        PERSISTENT_TOTAL_FLIGHTS: {
+            name: 'OSD_PERSISTENT_TOTAL_FLIGHTS',
+            text: 'osdTextTotalFlights',
+            desc: 'osdDescTotalFlights',
+            defaultPosition: -1,
+            positionable: true,
+            preview: "9876",
+        },
```

Comment:
Where is draw_order being used for?

Reply:
Is like z-index for osd elements. It must have a value accordingly with the order in the firmware.

Reply:
Limon, I think `draw_order` has to be `460` here, could you please add.

Reply:
I saw not all elements has it, so should be a default value, right?

Reply:
All the osd elements have this. If someone is missing we need to add them. This makes the preview more realistic when a user mounts one element over other.

Reply:
So far I've spotted these elements not having a `draw_order` value.
They are in the same function `OSD.loadDisplayFields = function() {};` but not necessary relate to the OSD elements in firmware?
```
TIMER
CPU_LOAD
VOLTAGE_WARNING
ARMED_TIME
ARMED
```
Not all occurrences using the value are placed in order. Only an observation and not part of this PR.

Reply:
For this PR i would suggest a value of `460`, but it's just my assumption.

Reply:
@haslinghuis  im not completely understanding how should i select the value. Could you please explain? Thanks

Reply:
If have no clue either - values seems arbitrary to me. Let's wait for @McGiverGim

Reply:
To check try the line (after line 1180):
```
draw_order: 460, 
```
and see if there is a difference?
But I guess this was obvious.

Reply:
yeah they are kinda following in the order as I can see. Thanks!

Reply:
The OSD elements are drawn in certain order in firmware:

https://github.com/betaflight/betaflight/blob/60c008b20bd025eadf0d7b7eb6a35c5e9f615756/src/main/osd/osd_elements.c#L1556-L1625

The OSD elements in the preview must be drawn in the same order. In your firmware PR you are adding it after `OSD_EFFICIENCY`, that has `draw_order: 455,` so you need to add some numbers to this. We usually add 5 to this number, maybe in the future we want to add some element between them. So 460 is a good number.

Usually this order does not matter, but some elements need to be over other. The crosshair must be over the horizon artificial, the warnings over all of the other osd elements, etc. Some users mount some elements over others to occupy less space. This gives them an accurate preview.

The elements that you have found and don't have preview are legacy elements that do not exists in the current firmwares.